File: programacao/Ex19.c
#include<stdio.h>
int main(int argc, char const *argv[])
{
    int a, v0, t, v;
    printf("Digite o Valor da aceleracao(m/s2):  ");
    scanf("%i", &a);

    printf("Digite o Valor da velocidade inicial(m/s): ");
    scanf("%i", &v0);

    printf("Digite o Tempo de percurso(s): ");
    scanf("%i", &t);

    v = v0 + (a*t);
     printf("A velocidade deu: %i km/h", v);

    if(v<=40){
        printf("\nVeiculo muito lento");
    }
    else if(v<=60){
        printf("\nVelocidade permitida");

    }
    else if(v<=80){
        printf("\nVelocidade de cruzeiro");
    }
    else if(v<=120){
        printf("\nVeiculo rapido");
    }
    else{
        printf("\nVeiculo muito rapido");
    }

    return 0;
}

File: python/Ex19.py
a = float(input("Digite o valor da aceleração(m/s2): "))
vi = float(input("Digite a velocidade inicial(m/s): "))
t = float(input("Digite o tempo de percurso(s): "))
vf = vi + a * t
if(vf <= 40):
    print("Veiculo muito lento")
else:
    if(vf <= 60):
        print("Veiculo em velocidade permitida")
    else:
        if(vf <= 80):
            print("Veiculo em velocidade de cruzeiro")
        else:
            if(vf <= 120):
                print("Veiculo rápido")
            else:
                print("Veiculo muito rápido")
